### ISSUE TITLE
Doesn't work for buttons created with $.ajax

### ISSUE THREAD

**Issue by qiangqiang101** [KajPe/bootstrapmodal#1]
Doesn't work for buttons created with $.ajax

**Comment by KajPe** [comment]
Yes, initially was for plain html so users don't have to script anything.
Don't know your use case, but added index2.html & modal2.js if it helps.

Basically it gets all elements which has classname 'showmodal' and adds an event to just show the modal. So you could also just add the addEventListener to your button and call showModal(modal-name).

**Comment by qiangqiang101** [comment]
button or a elements that were hard coded in html works, but if the elements were generated dynamically it doesn't work

I use datatables to load my data from database, which is fire on page load, click the edit button on tablecell nothing, the add button which is hard coded is working fine, both add and edit modal also hard coded, only the textvalues in the form is changed during button.onclick
